### PR TITLE
getopt-win32: add recipe for v0.1

### PR DESCRIPTION
**Pull request by nehaljwani**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by nehaljwani** [comment]
@conda-forge/help-c-cpp This is ready for review!

File: recipes/getopt-win32/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libimobiledevice-win32/getopt/archive/{{ version }}.tar.gz
+  sha256: 7e9653ecd58ce4149959bf6a905f4ab2f7889856fe1218afbf84284074f9e549 
+
+build:
+  number: 0
+  skip: true  # [not win]
```
Comment (isuruf) [inline recipes/getopt-win32/meta.yaml:14]: Add `run_exports` here?

Comment (nehaljwani) [reply recipes/getopt-win32/meta.yaml:14]: @isuruf I've not idea about the compatibility here. Should I put `x.x.x` ?

Comment (isuruf) [reply recipes/getopt-win32/meta.yaml:14]: Yes, that's a good idea.

**Pull request merged**